### PR TITLE
[android] Fix or suppress warnings

### PR DESCRIPTION
**Pull request by rtsisyk**
100+ warnings still left...

### CODE REVIEW

**Comment by arnaudvergnet** [comment]
Maybe we can leave that for now and work on it when fixing https://github.com/organicmaps/organicmaps/issues/4241#event-8177208827?

**Comment by Jean-BaptisteC** [comment]
#3721

File: android/src/app/organicmaps/widget/placepage/PlacePageBookmarkFragment.java [hunk 1]
```diff
@@ -104,7 +104,7 @@ else if (StringUtils.nativeIsHtml(notes))
     else
     {
       mTvBookmarkNote.setText(notes);
-      Linkify.addLinks(mTvBookmarkNote, Linkify.ALL);
+      Linkify.addLinks(mTvBookmarkNote, Linkify.WEB_URLS | Linkify.EMAIL_ADDRESSES | Linkify.PHONE_NUMBERS);
```
Comment (biodranik) [inline android/src/app/organicmaps/widget/placepage/PlacePageBookmarkFragment.java:107]: Эта штука раньше [дёргала вебвью](https://developer.android.com/reference/android/text/util/Linkify#MAP_ADDRESSES), интересно, может поэтому тоже крашило...

**Review by biodranik — APPROVED** [review]

**Comment by biodranik** [comment]
Всё пихать в utils конечно не самая хорошая идея. Код, помеченный "на местах использования" как deprecated быстрее прорефакторится...

**Comment by biodranik** [comment]
@rtsisyk 
> Unfortunately, there is no way to refactor code to work both on <31 and >=31 APIs at the same time. It looks like that Google intentionally breaks PackageManager API and adds new compat wrappers into their [proprietary library](https://developers.google.com/android/reference/com/google/android/gms/instantapps/PackageManagerCompat) to lock down developers...

Where do we need it? Why PackageManager can't be used instead?

**Pull request merged**